server/service/integration_core_test.go

```diff
@@ -2037,6 +2037,59 @@ func (s *integrationTestSuite) TestGlobalPoliciesAutomationConfig() {
 	require.Empty(t, config.WebhookSettings.FailingPoliciesWebhook.PolicyIDs)
 }
 
+func (s *integrationTestSuite) TestTeamsEndpointsWithoutLicense() {
+	t := s.T()
+
+	// list teams, none
+	var listResp listTeamsResponse
+	s.DoJSON("GET", "/api/v1/fleet/teams", nil, http.StatusPaymentRequired, &listResp)
+	assert.Len(t, listResp.Teams, 0)
+
+	// create team
+	var tmResp teamResponse
+	s.DoJSON("POST", "/api/v1/fleet/teams", &createTeamRequest{}, http.StatusPaymentRequired, &tmResp)
+	assert.Nil(t, tmResp.Team)
+
+	// modify team
+	s.DoJSON("PATCH", "/api/v1/fleet/teams/123", fleet.TeamPayload{}, http.StatusPaymentRequired, &tmResp)
+	assert.Nil(t, tmResp.Team)
+
+	// delete team
+	var delResp deleteTeamResponse
+	s.DoJSON("DELETE", "/api/v1/fleet/teams/123", nil, http.StatusPaymentRequired, &delResp)
+
+	// apply team specs - does succeed unlike others, no license required for this one
```

Comment:
Bug? (I'd say create a bug issue :)

Reply:
I'm really not sure? It seems apply specs could have a different approach, as I think it's called by fleetctl, but yeah if we think it's a bug I'll create the ticket.

Reply:
And I would leave the test as is so that it fails right away after the developer fixes the issue (and then the developer needs to amend the test).

Reply:
https://github.com/fleetdm/fleet/issues/3782